assets/src/scripts/models.js

```diff
@@ -34,8 +35,8 @@ function get(url, callback) {
 
 /**
  * Get a given timeseries dataset from Water Services.
- * @param  {Array}    options.sites  Array of site IDs to retrieve.
- * @param  {Array}    options.params List of parameter codes
```

Comment:
Ditto from above - I think we want to keep options here

Reply:
Agreed. It seemed like there were inconsistancies but it was probably inconsistancies that I introduced 😄